src/components/LinkedDatasourceView.tsx

```diff
@@ -12,13 +13,14 @@ interface Props {
 
 export const LinkedDatasourceView = ({ type }: Props) => {
   const navigate = useNavigation();
-  const { instance, meta } = useContext(InstanceContext);
+  const { instance } = useContext(InstanceContext);
+  const { jsonData } = useMeta();
   if (!instance.metrics || !instance.logs) {
     return <Spinner />;
   }
 
   const info = type === 'prometheus' ? instance.metrics : instance.logs;
-  const hostedId = type === 'prometheus' ? meta?.jsonData?.metrics.hostedId : meta?.jsonData?.logs.hostedId;
+  const hostedId = type === 'prometheus' ? jsonData?.metrics.hostedId : jsonData?.logs.hostedId;
```

Comment:
`jsonData` is a bit ambiguous and it's hard to understand what data it refers too. This refers to global settings, right? If so, what do you think of renaming it to something like that?

Reply:
The thing is that is what it is called in the provisioning file for the plugin so I have some reluctance to rename it. I like that if we leave it as `jsonData` we know that this is what is being provided in `provisioning/plugins/default.yaml` file.

I am actually tempted to go the other route and rename `GlobalSettings` to `PluginJsonData` 😅